src/client/js/Dialogs/PluginConfig/PluginConfigDialog.js

```diff
@@ -74,8 +76,21 @@ define(['js/Controls/PropertyGrid/PropertyGridWidgetManager',
         this._divContainer = this._dialog.find('.modal-body');
 
         this._saveConfigurationCb = this._dialog.find('.save-configuration');
+        this._modalHeader = this._dialog.find('.modal-header');
 
-        this._title = this._dialog.find('.modal-title');
+        if (typeof this._pluginMetadata.icon === 'string') {
```

Comment:
I would say that it is probably not a bad idea to allow extra classes for an icon file as it would give even more options. That would also makes it clear that there is no need for precedence.